etp-core/etp-backend/src/main/clj/solita/etp/service/aineisto.clj

```diff
@@ -103,7 +103,7 @@
         current-part (ByteBuffer/allocate (* 8 1024 1024))
         upload-parts-fn (fn [upload-part-fn]
                           (csv-reducible-query (fn [^String row]
-                                                 (let [row-bytes (.getBytes row (StandardCharsets/UTF_8))]
+                                                 (let [row-bytes (.getBytes row StandardCharsets/UTF_8)]
```

Comment:
Luulisi, että olisi ollut joku huomattava ero nyt generoitavan csv:n ja entisen välillä, mutta en sellaista huomannut.. 

Mitenköhän tämä toimi kuitenkin? 🤔

Reply:
Tuosta oli jossain maininta, ootas kun etin

Reply:
Ainakin täällä https://github.com/clj-kondo/clj-kondo/issues/2260

> The (Class/field) interop form works, but only accidentally and it's undocumented. Because of changes in clojure 1.12 it will no longer work and kondo (or lsp) should warn about it.

Reply:
Ja täällä https://insideclojure.org/2024/02/12/method-values/

> Static fields are another kind of qualified symbol, Aclass/staticField that already existed in Clojure. Static fields evaluate to their value and do not need to be invoked with parens. Due to details of the implementation, surrounding static fields with parens does appear to have the same effect, but this has not ever been published as valid syntax and it breaks the general substitutability of Clojure expressions - a static field in parens should invoke the value of the field.

> For backwards compatibility, this unintended behavior has been retained in Clojure 1.12, but clj-kondo will now warn about the use of (AClass/staticField) and we hope that eventually we can remove this provision in Clojure.